app/models/activity.rb

```diff
@@ -7,10 +7,19 @@ class Activity < ApplicationRecord
   validates :lesson_part_id, presence: true
   validates :duration, presence: true, numericality: { less_than_or_equal_to: 60 }
 
-  validates :default, inclusion: [true, false]
-  validates :default, uniqueness: { scope: :lesson_part_id }, if: :default?
+  attr_accessor :default
+
+  after_create :make_default!, if: :default
```

Comment:
This is only required _if_ we want to set the default at time of creation, otherwise we can create activities then promote one to be the default.

Reply:
I think this should be noted in the code